fuzz/fuzz_targets/channel_target.rs

```diff
@@ -200,8 +201,10 @@ pub fn do_test(data: &[u8]) {
 
 	let mut channel = if get_slice!(1)[0] != 0 {
 		let chan_value = slice_to_be24(get_slice!(3));
-
-		let mut chan = match Channel::new_outbound(&fee_est, chan_keys!(), their_pubkey, chan_value, slice_to_be24(get_slice!(3)), get_slice!(1)[0] == 0, slice_to_be64(get_slice!(8)), Arc::clone(&logger)) {
+		let push_msat = slice_to_be24(get_slice!(3));
```

Comment:
It'd be nice to avoid changing the order of data being loaded here as doing so will invalidate all existing fuzz test inputs collected.

Reply:
I didnt? Unless I am looking into this wrong. Because I didnt mean to. All I did was load the values into variables before sending to the function. The reason for this is so that annouce channel can be saved in the config file

Reply:
Oh, oops, my bad, I had managed to confuse myself into thinking chan_keys! reads some bytes.